iocore/net/P_QUICNetVConnection.h

```diff
@@ -145,6 +147,11 @@ class QUICNetVConnection : public UnixNetVConnection,
   void init(QUICConnectionId peer_cid, QUICConnectionId original_cid, QUICConnectionId first_cid, UDPConnection *,
             QUICPacketHandler *, QUICConnectionTable *ctable);
 
+  // QUICEventDriver
+  void set_write_event_pending(bool pend = true) override;
+  bool is_event_pending() const;
```

Comment:
Pending sending processing when credit is not enough. The generator should set pending flag when it is pended by credit. So the credit manager(CC, FC) can reenable write when credit frame coming (MAX_XXX_FRAME or ACK FRAME).